modules/external-load-balancer/main.tf

```diff
@@ -33,17 +33,162 @@ resource "google_compute_backend_service" "application" {
+    default_service = google_compute_backend_service.application.self_link
+    name            = "allpaths"
+
+    description = "Redirect requests to the application or the install dashboard path."
+    # This rule covers paths which are unique to the install dashboard.
+    path_rule {
+      # FIXME: https://tfe.emp-gcp.ptfedev.com/api/v1/support returns 502
+      paths = [
+        "/api/v1",
+        "/api/v1/*",
+        "/auditlog",
+        "/auditlog/*",
+        "/authenticate",
+        "/authenticate/*",
+        "/ceph",
+        "/ceph/*",
+        "/cluster",
+        "/cluster/*",
+        "/console",
+        "/console/*",
+        "/dashboard",
+        "/dashboard/*",
+        "/license",
+        "/license/*",
+        "/premkit",
+        "/premkit/*",
+        "/releases",
+        "/releases/*",
+        "/settings",
+        "/settings/*",
+        "/support",
+        "/support/*",
+      ]
+
+      service = google_compute_backend_service.install_dashboard.self_link
+    }
+    # This rule covers install dashboard asset paths under /assets. /assets is also used by application so paths must
+    # be listed explicitly.
+    path_rule {
+      paths = [
+        "/assets/03f50a3a6f195be73370626cc2887118.woff",
+        "/assets/36c91049d46f93c801176f06abb4b7f0.woff2",
+        "/assets/448c34a56d699c29117adc64c43affeb.woff2",
+        "/assets/50145685042b4df07a1fd19957275b81.ttf",
+        "/assets/629a55a7e793da068dc580d184cc0e31.ttf",
+        "/assets/8e976923f9d9c249cf32e75191cf797a.woff",
+        "/assets/a410ef2b4a38aa95291d109020a0b723.woff",
+        "/assets/af7ae505a9eed503f8b8e6982036873e.woff2",
+        "/assets/auditlog.main.23a3afa649a44fbe9fdb.js",
+        "/assets/authenticate.main.23a3afa649a44fbe9fdb.js",
+        "/assets/b06871f281fee6b241d60582ae9369b9.ttf",
+        "/assets/brand.23a3afa649a44fbe9fdb.css",
+        "/assets/c2d24dfda8a924bb6109080139a75698.woff2",
+        "/assets/c7dcce084c445260a266f92db56f5517.ttf",
+        "/assets/cluster.main.23a3afa649a44fbe9fdb.js",
+        "/assets/commons.23a3afa649a44fbe9fdb.css",
+        "/assets/commons.23a3afa649a44fbe9fdb.js",
+        "/assets/consolesettings.23a3afa649a44fbe9fdb.css",
+        "/assets/consolesettings.main.23a3afa649a44fbe9fdb.js",
+        "/assets/dashboard.23a3afa649a44fbe9fdb.css",
+        "/assets/dashboard.main.23a3afa649a44fbe9fdb.js",
+        "/assets/e18bbf611f2a2e43afc071aa2f4e1512.ttf",
+        "/assets/ec4dad2ea1198fed8332457c7778e7f1.woff2",
+        "/assets/fa2772327f55d8198301fdb8bcfc8158.woff",
+        "/assets/fee66e712a8a08eef5805a46892932ad.woff",
+        "/assets/licenseview.main.23a3afa649a44fbe9fdb.js",
+        "/assets/releases.main.23a3afa649a44fbe9fdb.js",
+        "/assets/settings.23a3afa649a44fbe9fdb.css",
+        "/assets/settings.main.23a3afa649a44fbe9fdb.js",
+        "/assets/snapshots.main.23a3afa649a44fbe9fdb.js",
+        "/assets/support.main.23a3afa649a44fbe9fdb.js",
```

Comment:
How was this generated and discovered? and how do we maintain this if replicated assets change?

Reply:
This list was manually generated; it's certainly not a scalable approach. :disappointed:

Reply:
a non scalable approach is fine in the short term as long as we have very explicit documentation on how to regenerate this manually.